bundles/org.openhab.binding.modbus.sbc/src/main/java/org/openhab/binding/modbus/sbc/internal/ALD1Registers.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.modbus.sbc.internal;
+
+import static org.openhab.io.transport.modbus.ModbusConstants.ValueType.*;
+
+import javax.measure.Unit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.library.unit.SmartHomeUnits;
+import org.openhab.io.transport.modbus.ModbusConstants;
+import org.openhab.io.transport.modbus.ModbusConstants.ValueType;
+
+/**
+ * The {@link ALD1Registers} is responsible for defining Modbus registers and their units.
+ *
+ * @author Fabian Wolter - Initial contribution
+ */
+@NonNullByDefault
+public enum ALD1Registers {
+    TOTAL_ENERGY(0.01f, 28, UINT32, SmartHomeUnits.KILOWATT_HOUR),
+    PARTIAL_ENERGY(0.01f, 30, UINT32, SmartHomeUnits.KILOWATT_HOUR),
+    FEEDING_BACK_ENERGY(0.01f, 30, UINT32, SmartHomeUnits.KILOWATT_HOUR),
```

Comment:
Same index as above?

Reply:
They are for distinct Thing types. I added a comment.